src/app/pages/camper/announcements/announcements.component.ts

```diff
@@ -7,15 +7,35 @@ import { AnnouncementsService } from "../../../service/announcements.service";
   styleUrls: ['./announcements.component.scss']
 })
 export class AnnouncementsComponent implements OnInit {
-
+  
+  private announcementlist;
+  
   constructor(
   	public annoService:AnnouncementsService
-  ) { }
+  ) {}
 
   ngOnInit() {
+
   	this.annoService.getSheet().subscribe(response => {
-  		console.log(response);
+      var annoucementArr = response.values; 
+
+      this.announcementlist = this.getLatest(annoucementArr);
+
   	})
   }
 
+  getLatest(arrList) {
+      var temp = [];
+
+      for (var i = arrList.length-1; i >= 0; i-- ){
+
+        if(temp[arrList[i][1]-1] == undefined )
```

Comment:
Just a code format I prefer... 
for binary operators (i.e. +, -, *, /) have a space between the operator and operand 

**Bad:** 
1+2;
a==b; 

**Good:**
1 + 2;
a == b;

for unary operator (i.e. ++, --), no spacing between operator and operand 

**Bad:**
i ++;
-- j;

**Good:**
i++;
--j;